examples/calculate_eventdisplay_sensitivity_cut2.py

```diff
@@ -0,0 +1,171 @@
+"""
+Example for using pyirf to calculate IRFS and sensitivity from EventDisplay DL2 fits
+files produced from the root output by this script:
+
```

Comment:
Probably better add a paragraph to explain the difference wrt to the main script